.circleci/config.yml

```diff
@@ -10,6 +10,12 @@ jobs:
       - run:
           name: Install required tools
           command: pacman -Sy --noconfirm base base-devel git python python-json5 python-pycryptodome
+      - run:
```

Comment:
Install with pacman like the rest of the packages, not pip.

Reply:
Just added yapf like this:
command: pacman -Sy --noconfirm base base-devel git python python-json5 python-pycryptodome yapf